wagtail/admin/panels/inline_panel.py

```diff
@@ -141,9 +154,23 @@ def __init__(self, **kwargs):
                 )
 
             empty_form = self.formset.empty_form
-            empty_form.fields[DELETION_FIELD_NAME].widget = forms.HiddenInput()
+            empty_form.fields[DELETION_FIELD_NAME].widget = forms.HiddenInput(
+                attrs={"data-w-formset-target": "deleteInput"}
+            )
             if self.formset.can_order:
-                empty_form.fields[ORDERING_FIELD_NAME].widget = forms.HiddenInput()
+                empty_form.fields[ORDERING_FIELD_NAME].widget = forms.HiddenInput(
+                    attrs={"data-w-formset-target": "orderInput"}
+                )
+
+            for field in self.formset.management_form:
+                if field.name.endswith(forms.formsets.TOTAL_FORM_COUNT):
+                    field.field.widget.attrs["data-w-formset-target"] = (
+                        "totalFormsInput"
+                    )
+                if field.name.endswith(forms.formsets.MIN_NUM_FORM_COUNT):
+                    field.field.widget.attrs["data-w-formset-target"] = "minFormsInput"
+                if field.name.endswith(forms.formsets.MAX_NUM_FORM_COUNT):
+                    field.field.widget.attrs["data-w-formset-target"] = "maxFormsInput"
```

Comment:
I am not 100% sure how, but I feel as though we can do better here and either somehow pull in the abstractions already provided from `wagtail/admin/forms/formsets.py` or pull out into a method within this class.

I am sure @laymonage  may have some suggestions.

Reply:
I looked into this a bit more, it turns out this is possible by passing a `formset` class in the `formset_opts` in `InlinePanel.get_form_options()`.